[aon_timer,dv] Testplan Update for AON Timer

ctopal opened the pull request:
This commit includes updates on the testplan of AON Timer.

Signed-off-by: Canberk Topal <ctopal@lowrisc.org>

cindychip commented on hw/ip/aon_timer/data/aon_timer_testplan.hjson:
```diff
@@ -12,22 +12,60 @@
+            - If we are changing WDOG_BARK_THOLD to be lower than the current WDOG_COUNT
+            watchdog timer should raise the interrupt `wdog_timer_bark` .
+            - If we are changing WDOG_BITE_THOLD to be lower than the current WDOG_COUNT
+            watchdog timer should trigger a reset request .
```
super nit: looks like line 43 and 45 has an extra space before `.` :)

cindychip commented on hw/ip/aon_timer/data/aon_timer_testplan.hjson:
```diff
@@ -12,22 +12,60 @@
+            - WKUP_COUNT should reflect expected passing of real time regardless of
+            prescaler value.
+            '''
+      milestone: V1
```
Just double check: Is this intended to be a V1 test? Same for wdog_lock test.

cindychip reviewed (commented):
Thanks for the PR :)
Looks great with a few questions. 
Also one more general questions: are these all the tests you plan to implement for aon_timer? I also saw `wake_cause` register, do you plan to check that value in some test?

ctopal commented:
Thanks for the review! I fixed the spacing problem and changed the milestone for stress test to be V2. For wdog_lock test, I think it's reasonably straightforward to check so I think might be better to put it into V1.

cindychip commented on hw/ip/aon_timer/data/aon_timer_testplan.hjson:
```diff
@@ -12,22 +12,60 @@
             **Checks**:
-            - TBD
+            - If we are changing WDOG_BARK_THOLD to be lower than the current WDOG_COUNT
+            watchdog timer should raise the interrupt `wdog_timer_bark` .
```
maybe I did not comment the right place, still see some extra space before the `.`
Same for line 25.

cindychip reviewed (approved):
LGTM with one more small nit.
